pages/overview.mdx

```diff
@@ -1,29 +1,111 @@
 # Technical Overview
+## What happens when you publish a site to Distributed Press?
+When you [upload content to a site](./api-reference/site#uploading-content-to-a-site) on Distributed Press, the server first 
+untars the files to a temporary folder on the server.
 
-Distributed Press has many components that work together.
+Then, the server looks up the associated configuration for the site. It figures out what 
+protocols the user has indicated interest in publishing to and then kicks off sync tasks
+for the enabled protocols.
 
-## API Server
-The API server is how clients interface with Distributed Press. 
+### IPFS
+For IPFS, the associated logic is [in this file](https://github.com/hyphacoop/api.distributed.press/blob/main/protocols/ipfs.ts).
+The main things you need to know is that Distributed Press keeps a long-running [Kubo](https://www.npmjs.com/package/go-ipfs)
+instance that is spun-up when Distributed Press starts up. This connection is managed by `js-ipfsd-ctl`.
 
-Clients make authenticated HTTP requests to make updates to resources 
-like publishers, admins, and sites.
+We then copy the files to an IPFS [Mutable File System (MFS)](https://docs.ipfs.tech/concepts/file-systems/) and 
+publish it to IPFS. When finished, this returns us a CID. We update the IPNS entry for the site ID to this resulting CID.
 
-The API server is responsible for locally storing site data to serve and 
-seed over the various [peer-to-peer protocols](./protocols) that Distributed Press supports.
+This CID and accompanying IPNS entry is used to populate the `links` section:
```

Comment:
The links section of what? Is there an example / reference config file you can point to?

Reply:
THe `links` section when you do a `GET /v1/sites/{your site domain/`.

E.g. this is what my blog looks like:

```
{
  "id": "blog.mauve.moe",
  "domain": "blog.mauve.moe",
  "protocols": {
    "http": true,
    "hyper": true,
    "ipfs": true,
    "bittorrent": true
  },
  "links": {
    "http": {
      "enabled": true,
      "link": "example-link"
    },
    "hyper": {
      "gateway": "https://blog-mauve-moe.hyper.hypha.coop/",
      "raw": "hyper://1m51x54k3fwbuec5z4edbathiq3aj6bew8b556ezrszizskogo3o/",
      "dnslink": "/hyper/1m51x54k3fwbuec5z4edbathiq3aj6bew8b556ezrszizskogo3o/",
      "enabled": true,
      "link": "hyper://blog.mauve.moe"
    },
    "ipfs": {
      "gateway": "https://blog-mauve-moe.ipns.ipfs.hypha.coop",
      "cid": "bafybeidpngig3g4gqrbooyrzxe4vl5jusad7om2chtl7vgrzcqbpilni5e",
      "pubKey": "ipns://k51qzi5uqu5dgxx5s83cvbehacxojuztwxnhnmxm1npizgve9e3fehpk2lqnf9/",
      "dnslink": "/ipns/k51qzi5uqu5dgxx5s83cvbehacxojuztwxnhnmxm1npizgve9e3fehpk2lqnf9/",
      "enabled": true,
      "link": "ipns://blog.mauve.moe/"
    },
    "bittorrent": {
      "gateway": "https://blog-mauve-moe.bt.hypha.coop/",
      "magnet": "magnet:?xt:urn:btih:bbad2f8321989a8395a6b1bac6240877ab86727a&xs=urn:btpk:416ce4db34ae1260d81c2aed16bd1eab3481e4e48f3c781aa9c057b15d900b71",
      "infoHash": "bittorrent://bbad2f8321989a8395a6b1bac6240877ab86727a/",
      "pubKey": "bittorrent://416ce4db34ae1260d81c2aed16bd1eab3481e4e48f3c781aa9c057b15d900b71/",
      "dnslink": "/bittorrent/416ce4db34ae1260d81c2aed16bd1eab3481e4e48f3c781aa9c057b15d900b71",
      "enabled": true,
      "link": "bittorent://blog.mauve.moe/"
    }
  },
  "public": true
}
```